### ISSUE TITLE
install.sh prefers copying files into oldest version of Photoshop instead of newest

### ISSUE THREAD

**Issue by xaphod** [jessesquires/iOS-icons-script#8]
Issue: install.sh copies its files into only one folder that it finds (of CC 2019 and 2020). That's the wrong one by default: CC 2019, when 2020 exists, so Script is missing.

Suggestion: copy into all folders found (not just first one), because by default, Adobe does NOT delete the old folder when "uninstalling" the older versions of photoshop -- they're left around in case you want to reinstall later and keep your settings.

Alternate suggestion: reverse the order of VERSIONS so that 2020 is searched first.

**Comment by xaphod** [comment]
Also, I love the "Fuck yeah" at the end. Perfect

**Comment by jessesquires** [comment]
thanks @xaphod !

pull requests welcome 😄

**Comment by greghaygood** [comment]
Hey @jessesquires! I stumbled on this script and it's just what I needed. Then I ran into the same issue as @xaphod. Is the preference to install into all compatible versions, or just the most recent? I've updated the script to handle the case of all of them, since that would seem to be more helpful to more folks than picking a version (or having the script prompt for each version). Just lemme know your preference and I can submit the PR. Thanks!
